appcontainer/Dockerfile

```diff
@@ -52,6 +52,23 @@ RUN useradd --create-home --shell /bin/bash $USER && \
     apt-get install -qq --no-install-recommends build-essential nginx gettext && \
     python -m pip install --upgrade pip
```

Comment:
Let's take a follow-up to add the Buildkit cache mounting feature to these Dockerfiles, for `apt`, `pip`, etc.

Reply:
Perfect, I was also thinking that I should add this feature soon 😄